cpp/src/slice2cs/Gen.cpp

```diff
@@ -800,7 +739,8 @@ Slice::CsVisitor::writeParameterDocComments(const DocCommentPtr& comment, const
         auto q = commentParameters.find(param->name());
         if (q != commentParameters.end())
         {
-            _out << nl << "/// <param name=\"" << fixId(param->name()) << "\">";
+            string fixedParamName = addPrefixToIdentifier("", param->mappedName()); // Strip off any leading '@'s.
+            _out << nl << "/// <param name=\"" << fixedParamName << "\">";
```

Comment:
I checked and doc-comment links to parameters _cannot_ start with `@`.
The link should just be to the actual identifier, without any escaping prefixes.